index.html

```diff
@@ -44,26 +44,40 @@
           <li>Services</li>
         </ul>
       </div>
+    </header>
+    <section class="homesection">
+      
+      
 
       <!-- NAVBAR SECTION  -->
       <div class="navbar">
         <img src="./images/icons/Union.svg" alt="Union img" id="union" />
         <div class="container">
+          <div class="containercontent">
           <a href="/index.html"
             ><img src="./images/icons/homeicon.jpeg" alt="home icon" id="homeicon"
           /></a>
```

Comment:
- [ ] This link still shows error: 

![image](https://user-images.githubusercontent.com/77038610/147338680-b1b5c4c1-b482-404d-83be-53072a7fe5d1.png)

This can be easily fixed if you add dot slash at the beginning of path from path like you did it for `about` link : `<a href="./index.html" >`